daemon/cmd/daemon_main.go

```diff
@@ -1085,8 +1089,22 @@ func initEnv(cmd *cobra.Command) {
 		}
 	}
 
+	checkHostFirewallWithEgressLB()
+
 	initKubeProxyReplacementOptions()
 
+	if option.Config.EnableHostFirewall && option.Config.Device == "undefined" {
+		device, err := linuxdatapath.NodeDeviceNameWithDefaultRoute()
+		if err != nil {
+			msg := "Host firewall's external facing device could not be determined. Use --device to specify."
```

Comment:
`... "User --%s to specify",  option.Device)`

Reply:
The option is `--device=` here ;-)